worker/migrations/migrate-accounts-to-organisation.go

```diff
@@ -201,31 +222,10 @@ func migrateAccountsToOrganization(domain string) error {
             errm = multierror.Append(errm, err)
         }
     }
-    // Reload the setting in case the revision changed
-    setting, err = settings.Get(inst)
-    if err != nil {
+
+    err = updateSettings(inst, 0)
+    if (err != nil) {
```

Comment:
There is no parenthesis in if statements in go ;)

Reply:
It's been removed in later commits.